espaloma/nn/readout/janossy.py

```diff
@@ -320,7 +320,7 @@ def forward(self, g):
 
 class JanossyPoolingNonbonded(torch.nn.Module):
     """Janossy pooling (arXiv:1811.01900) to average node representation
-    for improper torsions.
+    for nonbonded interactions.
 
 
     """
```

Comment:
@yuanqing-wang : Is this for 1-4 nonbonded exceptions, or for single-particle Nonbonded parameters?

Reply:
This is supposed to be both. But we did not incorporate this in the current version of scripts.